packages/lumx-react/src/components/side-navigation/SideNavigationItem.tsx

```diff
@@ -94,6 +98,7 @@ const SideNavigationItem: React.FC<SideNavigationItemProps> = (props) => {
                 tabIndex={0}
                 onClick={onClick}
                 onKeyDown={onClick ? onEnterPressed(onClick as Callback) : undefined}
+                {...(linkProps || {})}
```

Comment:
I think `{...(linkProps || {})}` should appear in first position to make sure it does not override the other props on `the <a>` element